swagger.yml

```diff
@@ -79,13 +113,24 @@ paths:
         - in: query
           name: delivery_at
           type: string
-          required: true
+          default: "Today"
```

Comment:
May be use some other way of surrounding the word not to confuse with default string value? Like _{Today}_

Reply:
I would expect something like to exsist but I couldn't find it...